ci/get_labels/main.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"net/http"
+	"strings"
+)
+
+var prUrl = flag.String("url", "", "Url of pull request")
+
+type respLabels struct {
+	Name string
+}
+
+func main() {
+	flag.Parse()
+
+	labelCollector := LabelCollector{Client: &http.Client{}}
+	fmt.Println(labelCollector.GetLabels(*prUrl))
+}
+
+type LabelCollector struct {
+	Client *http.Client
+}
+
+func (l *LabelCollector) GetLabels(prUrl string) string {
```

Comment:
Optional: If I were writing this, I would have this function return an error, then `log.Fatal(err)` in main if it's not nil. Two reasons:

* I'm in favour of `log.Fatal` over a panic for a program like this that's more of a script, the panic rarely provides useful info
* I think it's nice to have the `main` function be the only one in charge of exiting the program

But what you have here is fine, I think you can leave it as is if you'd like. (I did already technically approve this part before so it's obviously fine enough lol)